scripts/lib/sphinx/sphinxHtmlToMarkdown.test.ts

```diff
@@ -1470,6 +1470,57 @@ test("identify release notes", async () => {
 `);
 });
 
+test("test HTML tag without id", async () => {
```

Comment:
Maybe add a comment linking to the original issue. It's hard to know what the problem is here since there is so much HTML.

Some test authors will even put the issue # explicitly in the test description.